class_defines.py

```diff
@@ -2180,7 +2192,7 @@ class SlvsAngle(GenericConstraint, PropertyGroup):
     value: FloatProperty(
         name=label, subtype="ANGLE", unit="ROTATION", update=update_system_cb
     )
-    setting: BoolProperty(name="Invert", update=update_system_cb)
+    setting: BoolProperty(name="Invert", get=invert_angle_getter, set=invert_angle_setter)
```

Comment:
"setting" name is quite confusing. Maybe explicitly name it inverted or should_invert or similar?

Reply:
True, but I left it alone since the Distance constraint has "setting" that means "Flip."  There is shared infrastructure.  The parent class uses "setting" without knowing what it means.   Having a more descriptive name would require some re-engineering.